docs/platforms/javascript/common/best-practices/micro-frontends.mdx

```diff
@@ -192,3 +192,17 @@ captureException(new Error("oh no!"), (scope) => {
   return scope;
 });
 ```
+
+## SDK Version Alignment
+
+Starting with version 8, if you have multiple Sentry JavaScript SDKs on the same page, they will [only interact with each other](https://github.com/getsentry/sentry-javascript/pull/12206) if they're all using the same version. This prevents unwanted cross-SDK interactions, for example, from browser extensions or 3rd party scripts.
+
+However, for micro frontends, where you _want_ SDK interaction between multiple SDKs, you'll need to ensure that all SDKs are using the same version.
+
+Version alignment is important when you initialize the SDK in a host or skeleton application, but use Sentry SDK calls in micro frontend child applications. For example, if your individual micro frontends call `Sentry.captureException`.
```

Comment:
Might be worth adding something like "Module federation is one way to achieve version alignment."

Reply:
I'm by far no module federation expert but to me it module federation is one of many ways how to architect an MFE setup. How would module federation guarantee version alignment?

Reply:
The aspect of Module Federation that I was thinking about was the dependency sharing. Perhaps it could be said more generically. 

But the idea is: if the `@sentry/browser` dependency is _shared_ (amongst the host and MFEs), then it will necessarily be the same version. (Of course, if the MFE doesn't make use of that dependency and chooses to use its own version, that's another story).